MatrixPilot/flightplan-logo.c

```diff
@@ -637,6 +638,14 @@ static int16_t logo_value_for_identifier(uint8_t ident)
 		case WIND_SPEED_Z: // in cm/s
 			return estimatedWind[2];
 
+		case WIND_FROM_ANGLE: // wind from in degrees 0-359, 0 = North
+		{
+			int16_t angle = get_angle_to_point(estimatedWind[0], estimatedWind[1]);
+			while (angle < 0) angle += 360;

```

Comment:
I think these while loops should just be if blocks, unless there's a way get_angle_to_point() can return under -360 or over 720.

Reply:
Ben,

Ok, i see that most of the similar code uses if's, i'll change that. Thanks. 
  
Best regards,
Kees